integration/kubernetes/confidential/agent_image.bats

```diff
@@ -31,7 +31,17 @@ RUNTIMECLASS="${RUNTIMECLASS:-kata}"
 test_tag="[cc][agent][kubernetes][containerd]"
 
 setup() {
-	setup_common
+	remove_test_image "$image_unsigned_protected" || true
```

Comment:
So as described in above, we may should remove all images tested in the following test cases like this: 
```
 remove_test_image "$image_cosigned" || true
 remove_test_image "$image_cosigned_other" || true
 remove_test_image "$image_simple_signed" || true
 remove_test_image "$image_signed_protected_other" || true
 remove_test_image "$image_unsigned_protected" || true
 remove_test_image "$image_unsigned_unprotected" || true
 remove_test_image "$image_authenticated" || true

```